src/app/reporting/reporting.cpp

```diff
@@ -354,17 +365,27 @@ void emberAfPluginReportingTickEventHandler(void)
 
     if (apsFrame != NULL)
     {
-        conditionallySendReport(apsFrame->sourceEndpoint, apsFrame->clusterId);
+        conditionallySendReport(apsFrame->sourceEndpoint, apsFrame->clusterId, reportDestination, destinationEndpoint);
```

Comment:
I'm a little confused about why there is another `if (apsFrame != NULL)` test, is it able to move this statement inside `if (apsFrame == NULL)` branch ?